keylime.conf

```diff
@@ -110,6 +110,16 @@ revocation_actions=
 # with a working directory of /var/lib/keylime/secure/unzipped.
 payload_script=autorun.sh
 
+# The path to the directory containing the pre-installed revocation action
+# scripts.  Ideally should point to an fixed/immutable location subject to
+# attestation.  The default is /usr/libexec/keylime.
+revocation_actions_dir = /usr/libexec/keylime
```

Comment:
I'm curious why these are needed. It looks like it's just setting them as the default. Does the rust agent not use the same defaults?

Reply:
@mpeters It's based on [enhancement 55](https://github.com/keylime/enhancements/blob/master/55_revocation_actions_without_python.md#design-details), if you look at the design details. Not sure if there is already an issue open to implement all of the changes in that doc, if so then I could link it. The other parts of the enhancement are still to-do, but we can't get this [PR](https://github.com/keylime/rust-keylime/pull/321) through on the Rust agent until these config options are added because it expects them and throws an error otherwise.

This is actually part of a larger meta-discussion that could be had on how to keep the two repos in sync for changes like this.

If it would be too disorganized to add this to `keylime/keylime`, we can probably do a workaround on the Rust agent side instead. But if this change needs to be made to `keylime/keylime` anyway eventually I figured I would go ahead and add it.

Reply:
@mpeters When I wrote the changes to the rust agent I did not add hardcoded defaults and expected the values to always be present in the configuration file, which now seems to me a naive approach.

I think the best here is to add hardcoded default values so that the rust agent is able to run even when the values are not present in the configuration file. I'll make this change to the [PR](https://github.com/keylime/rust-keylime/pull/321).

This doesn't solve the problem of having the configuration files out of sync, though.